src/npcmove.cpp

```diff
@@ -4378,6 +4382,17 @@ void npc::do_reload( const item &it )
 
 bool npc::adjust_worn()
 {
+    bool any_broken = false;
+    for( int i = 0; i < num_hp_parts; i++ ) {
+        if( hp_cur[i] <= 0 ) {
```

Comment:
Please use character::is_limb_broken instead of checking the hp.  Thanks

Reply:
done, thanks